cmd/root.go

```diff
@@ -16,18 +16,19 @@ import (
 
 //define flags in here
 var (
-	repo       string
-	format     string
-	skipGitCLI bool
-	cui        bool
+	repo         string
+	format       string
+	skipGitCLI   bool
+	cui          bool
+	defaultQuery = 0
 )
 
 func init() {
 	rootCmd.PersistentFlags().StringVar(&repo, "repo", ".", "path to git repository (defaults to current directory). A remote repo may be specified, it will be cloned to a temporary directory before query execution.")
 	rootCmd.PersistentFlags().StringVar(&format, "format", "table", "specify the output format. Options are 'csv' 'tsv' 'table' and 'json'")
 	rootCmd.PersistentFlags().BoolVar(&skipGitCLI, "skip-git-cli", false, "whether to *not* use the locally installed git command (if it's available). Defaults to false.")
 	rootCmd.PersistentFlags().BoolVarP(&cui, "interactive", "i", false, "whether to run in interacive mode, which displays a terminal UI")
-
+	rootCmd.PersistentFlags().IntVar(&defaultQuery, "query", 0, "used to pick a default query")
```

Comment:
I think it would be a better experience to have the query specified by a shortname, rather than an index/number. It'd mean the other preset queries will need to be "named", but I think it's worthwhile: `all_commits`, `distinct_authors`, `distinct_author_emails`, `count_distinct_author_emails` etc